examples/megacli.cpp

```diff
@@ -4395,15 +4403,15 @@ void exec_sync(autocomplete::ACState& s)
     }
     else if (s.words.size() == 2)
     {
-        int i = 0, cancel = atoi(s.words[1].s.c_str());
+        handle cancelBackupId = atoll(s.words[1].s.c_str());
```

Comment:
Input param should be in B64 rather than binary represented as long integer. I'll adjust it.

Reply:
Done